segment_protocols/loan-clones.md

```diff
@@ -0,0 +1,37 @@
+# LOAN clones protocol
+
+This is a list of all known 'clone' accounts. These are users who are running open source bots on their accounts. With aditional information
```

Comment:
Spelling? :) `additional`

Reply:
I copied it from the .json file you gave me :P The spelling seems fine to me? :)